src/views/OrderDetail.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <ion-page>
+    <ion-header>
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start" />
+        <ion-title>{{ $t("Order Details") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+      <div>
+      <ion-list>
+        <ion-item lines="none">
+          <ion-label>
+            <h2>{{order.customer?.first_name}} {{ order.customer?.last_name }}</h2>
+            <p>{{ order.name }}</p>
+          </ion-label>
+          <ion-badge color="dark" slot="end">{{ order.created_at }}</ion-badge>
+        </ion-item>
+      </ion-list>
+      <ion-item v-if="order.customer?.phone">
+        <ion-icon :icon="callOutline" slot="start" />
+        <ion-label>{{ order.customer?.phone }}</ion-label>
+        <ion-button
+          fill="outline"
+          slot="end"
+          color="medium"
+        >
+          {{ $t("Copy") }}
+        </ion-button>
+      </ion-item>
+      <ion-item lines="none" v-if="order.customer?.email">
+        <ion-icon :icon="mailOutline" slot="start" />
+        <ion-label>{{ order.customer?.email }}</ion-label>
+        <ion-button
+          fill="outline"
+          slot="end"
+          color="medium"
+        >
+          {{ $t("Copy") }}
+        </ion-button>
+      </ion-item>
+  
+      <main v-for="item in order.line_items" :key="item">
```

Comment:
Use a unique value as a key instead of using the whole object.

Reply:
Used a unique value as key.